Dockerfile

```diff
@@ -74,7 +74,7 @@ RUN case ${TARGETPLATFORM} in \
     "linux/arm64")  exit 1 ;; \
     *)              /opt/conda/bin/conda update -y conda &&  \
     /opt/conda/bin/conda install -y "python=3.9" && \
-    /opt/conda/bin/pip install torch==2.0.0+cu118 torchvision==0.15.1+cu118 torchaudio==2.0.1 --index-url https://download.pytorch.org/whl/cu118 ;; \
+    /opt/conda/bin/pip install torch==2.1.1+cu118 torchvision==0.15.1+cu118 torchaudio==2.0.1 --index-url https://download.pytorch.org/whl/cu118 ;; \
```

Comment:
torchaudio and torchvision versions are out of sync, but I don't think they're needed anyway. I'll land as-is and remove these in a follow-up.